projects/plugins/backup/src/js/components/admin-style.scss

```
@@ -128,3 +129,7 @@
 .jp-dashboard-footer {
 	padding: 40px 0;
 }
+
+.jp-connection-status-card--status {
+	margin: 24px 0;
+}
```

```suggestion
}

```

Let's add a trailing endline.